src/Components/PolicyAuthorization.cs

```diff
@@ -0,0 +1,33 @@
+using Microsoft.AspNetCore.Authorization;
+
+namespace IdentityServerConfig.Components;
```

Comment:
Perhaps this shouldn't be in the `Components` namespace (and folder) as it doesn't necessarily have anything to do with the components. Maybe this should just be in the root namespace.

Reply:
changed in adf529e